sw/device/lib/dif/dif_entropy.h

```diff
@@ -363,6 +363,14 @@ typedef enum dif_entropy_result {
    * succeed until hardware reset.
    */
   kDifEntropyLocked = 3,
+  /**
+   * Indicates that entropy is not yet available for software consumption
+   */
+  kDifEntropyDataUnAvailable = 4,
+  /**
+   * Indicates that entropy is not idle
+   */
+  kDifEntropyNotIdle = 5,
```

Comment:
Is there a reason this needs to be an error...?

Reply:
see above.